Apps/Src/SendTritium.c

```diff
@@ -0,0 +1,28 @@
+#include "SendTritium.h"
+#include "CarState.h"
+#include <math.h>
+
+
+static float velocity_to_rpm(float velocity) {
```

Comment:
Is the naming convention for static methods snake case? No preference from me really, just make sure it's consistent.

Reply:
I'm fairly sure that it's the convention that we've mostly been using. At the end of the day it doesn't really matter, the function isn't accessible outside of this file.